app/routes/_auth.signup.tsx

```diff
@@ -158,12 +159,12 @@ export default function Component() {
               </p>
             )}
 
-            <button type="submit" disabled={isPending}>
+            <AuthButton variant="primary" buttonStyle="fullyRounded" disabled={isPending}>
```

Comment:
What exactly was the reason for the `AuthButton` component (instead of using `Button`)? :)

Reply:
Mostly, I did it because of w-full and sm:w-fit to make it responsive. It didn't work well for auth pages because of the limited container size in login and sign-up pages. We can make it w-full all the time and make it responsive with the parent HTML element!